src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { View, loc } from 'okta';
+import Enums from 'util/Enums';
+import { WIDGET_FOOTER_CLASS } from '../../utils/Constants';
+
+const HCAPTCHA_URL = 
+  'https://hcaptcha.com/1/api.js?onload=OktaSignInWidgetOnCaptchaLoaded&render=explicit';
+const RECAPTCHAV2_URL = 
+  'https://www.google.com/recaptcha/api.js?onload=OktaSignInWidgetOnCaptchaLoaded&render=explicit';
+
+export default View.extend({
+  className: 'captcha-view',
+
+  initialize() {
+    if (this.options.appState.get('captcha')) {
+      this.captchaConfig = this.options.appState.get('captcha');
+      this._addCaptcha();
+    }
+  },
+
+  remove: function(){
+    View.prototype.remove.apply(this, arguments);
+
+    // Cleanup global Captcha references
+    if (this.captchaConfig.type === 'HCAPTCHA') {
+      window.hcaptcha = undefined;
```

Comment:
Is it possible that this may interfere with another `hcaptcha` on the `window` object? Instead, consider nesting this within the `window.okta` namespace - something we do with other required global attributes.

Reply:
So in this case it's the Captcha libraries that actually attach that references to the window object, and our intent here is to clean that up. Collisions are a potential concern and this is something we've discussed during the design (like what if the customer is using these Captcha libs already). The summary is that for now we don't support this as we will need to consider other factors like different versions as well. This is something that we will look at supporting as a future improvement potentially.